server/controllers/profile.js

```diff
@@ -5,14 +5,24 @@ const asyncHandler = require("express-async-handler");
 // @desc Create new profile
 // @access Public
 exports.createProfile = asyncHandler(async (req, res, next) => {
-  const { firstName, lastName, gender, birthDate, email, phoneNumber, location, profilePic, description, availability } = req.body;
-
   try {
```

Comment:
While try/catch is a good idea here, we have a library called `express-async-handler` that will catch thrown errors and respond with the json output. Which means we don't need this in our controller logic unless we are catching for a specific thrown error to provide context in a specific error that is expected.

Reply:
Where ever applicable please address by removing try/catch in your changes.

Reply:
noted. Thank you very much!